src/current/_includes/releases/v24.3/v24.3.0-alpha.1.md

```diff
@@ -1,3 +1,340 @@
+- It is now possible to create [PL/pgSQL]({% link v24.3/plpgsql.md %}) trigger functions, which can be executed by a trigger in response to table mutation events. Note that this patch does not add support for triggers, only trigger functions. [#126734][#126734]
+- Cluster settings [`enterprise.license`]({% link v24.3/cluster-settings.md %}#setting-enterprise-license) and [`diagnostics.reporting.enabled`]({% link v24.3/cluster-settings.md %}#setting-diagnostics-reporting-enabled) now have additional validation. [#131097][#131097]
+- The [`SHOW SESSIONS`]({% link v24.3/show-sessions.md %}) command was changed to include an `authentication_method` column in the result. This column will show the method used to authenticate the session, for example, `password`, `cert`, `LDAP`, etc. [#131625][#131625]
+
+<h3 id="v24-3-0-alpha-1-operational-changes">Operational changes</h3>
+
+- [Events]({% link v24.3/eventlog.md %}) `DiskSlownessDetected` and `DiskSlownessCleared` are now logged when disk slowness is detected and cleared on a store. [#127025][#127025]
+- Several [cluster settings]({% link v24.3/cluster-settings.md %}) allow you to configure rate-limiting traffic to cloud storage over various protocols. These settings begin with `cloudstorage`. [#127207][#127207]
+- The new [cluster setting]({% link v24.3/cluster-settings.md %}) `kv.range.range_size_hard_cap` allows you to limit how large a [range]({% link v24.3/architecture/overview.md %}#architecture-range) can grow before [backpressure]({% link v24.3/common-errors.md %}#split-failed-while-applying-backpressure-are-rows-updated-in-a-tight-loop) is applied. This can help to mitigate against a situation where a range cannot be split, such as when a range is comprised of a single key due to an issue with the schema or workload pattern or a bug in client application code. The default is 8 GiB, which is 16 times the default max range size. If you have changed the max range size, you may need to adjust this cluster setting or reduce the range size. [#129450][#129450]
+- The following `kvflowcontrol` [metrics]({% link v24.3/metrics.md %}) have been renamed. After a cluster is finalized on v24.3, old and new metrics will be populated. The previous metrics under `kvasdmission.flow_controller` will be removed.
+
+      Old metric names                                                     | New metric names
+      ---------------------------------------------------------------------|-----------------
+      `kvadmission.flow_controller.(regular|elastic)_tokens_available`     | `kvflowcontrol.tokens.eval.(regular|elastic).available`
```

Comment:
@mdlinville I can't get this to render.

Reply:
From my testing, you can either have
(1) a pipe character by using `&#124;` or  &vert; WITHOUT the backticks for code formatting
OR
(2) have the backticks for code formatting AND replace the pipe with another character, like a forward slash

``` 
      Old metric names                                                     | New metric names
      ---------------------------------------------------------------------|-----------------
      kvadmission.flow_controller.(regular&#124;elastic)_tokens_available     | kvflowcontrol.tokens.eval.(regular&vert;elastic).available
      `kvadmission.flow_controller.(regular/elastic)_tokens_deducted`      | `kvflowcontrol.tokens.eval.(regular/elastic).deducted`
```

Reply:
Thanks for showing me that workaround! I could also have wrapped the pipes within `{% raw %}...{% endraw %}`. I thought I was safe within backticks. Actrually I may have been, it looks like I also did not properly close the first cell in the table for example. In this situation the best solution is to fully-qualify the old and new metric names.